test/uk/gov/hmrc/helptosavestub/controllers/DWPControllerSpec.scala

```diff
@@ -43,22 +42,22 @@ class DWPControllerSpec extends UnitSpec with WithFakeApplication {
   "dwpEligibilityCheck" must {
 
     "return a 400 status with no json payload when given a nino starting with WS400" in {
-      val result: Future[Result] = dwpController.dwpClaimantCheck("WS400123A", systemId, threshold, Some(newUUID))(fakeRequest)
+      val result: Future[Result] = dwpController.dwpClaimantCheck(randomNINO.withPrefixReplace("WS400"), systemId, threshold, Some(newUUID))(fakeRequest)
```

Comment:
`randomNINO()` is declared with parentheses but called without here (and in other occurences in this file)